README.md

```diff
@@ -31,6 +31,7 @@ Main Features
 - Multi-line commands
 - Special-character command shortcuts (beyond cmd's `?` and `!`)
 - Command aliasing similar to bash `alias` command
+- Macros, which are similar to aliases, but can take arguments when called
```

Comment:
Pseudo-random train of thought ... we also have a similar feature list in **setup.py** which is what gets published to PyPI that I don't think we have updated in a while.  You should update that as well and make sure it is consistent.

Reply:
Done

Reply:
I'm not aware of a way to link them when using Markdown other than writing Python code in setup.py to parse the README.md file and pull out the lines we want.  Actually, this is probably totally do-able and a good idea.

If using reStructuredText, there are some other things you can do since it is more powerful and flexible than Markdown, but I would strongly prefer to keep the README in Markdown format due to the simplicity and lower learning curve.